lib/config.js

```diff
@@ -8,33 +8,27 @@ const GLOBAL_CONFIG = Symbol('globalConfig');
 const PROJECT_CONFIG = Symbol('projectConfig');
 const LOCAL_CONFIG = Symbol('localConfig');
 
-exports.GLOBAL_CONFIG = GLOBAL_CONFIG;
-exports.PROJECT_CONFIG = PROJECT_CONFIG;
-exports.LOCAL_CONFIG = LOCAL_CONFIG;
-
 function getNcurcPath() {
-  if (process.env.XDG_CONFIG_HOME !== 'undefined' &&
```

Comment:
I don't quite remember why we have this condition now...but I guess it doesn't hurt to keep it?